sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -50,7 +50,7 @@
     "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist esm test-dist typings *.tgz *.log",
     "extract-api": "tsc -p . && api-extractor run --local",
-    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"*.{js,json}\"",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"samples/*.ts\" \"*.{js,json}\"",
```

Comment:
You may want to go ahead and build the samples at the same time as the library, just to ensure that the samples continue to work as we do PRs in the future.

Reply:
Done!